CHANGELOG.md

```diff
@@ -92,6 +92,12 @@ External tooling making use of `searchContext` in the `GET` `/granules/` endpoin
   - Updated `@cumulus/api/bin/serveUtils` to no longer add records to ElasticSearch
   - Removed ElasticSearch from local API server code
   - Updated CollectionSearch to filter granule fields in addition to time frame for active collections
+- **CUMULUS-3847**
+  - remove remaining ES indexing in code and tests
+  - for asyncOperations test data, change any ES related values to other options
+  - remove code from `@cumulus/api/lambdas/cleanExecutions` leaving a dummy handler, as the code worked with ES. lambda will be rewritten with CUMULUS-3982
+  - remove `@cumulus/api/endpoints/elasticsearch`, `@cumulus/api/lambdas/bootstrap`, and `@cumulus/api/lambdas/index-from-database`
+  - clarify in comments that ES functionality remaining in `@cumulus/api/lib/granules` is for working with CloudMetrics
```

Comment:
Probably don't need this in the top-level release notes. The comments in the file (assuming they're there) are fine.